src/routes/Apps/App/TrainDialogs.tsx

```diff
@@ -1046,13 +1072,13 @@ class TrainDialogs extends React.Component<Props, ComponentState> {
     // Import a .transcript tile
     async onImport(transcript: BB.Activity[]): Promise<void> {
 
-        this.setState({isImportWaitModalOpen: true})
+        this.setState({ isImportWaitModalOpen: true })
 
         const transcriptHash = Util.hashText(JSON.stringify(transcript))
 
         // If transcript has already been imported, go to next one
         if (this.hasTranscriptBeenImported(transcriptHash)) {
-            this.setState({isImportWaitModalOpen: false})
+            this.setState({ isImportWaitModalOpen: false })
```

Comment:
By the way, what do you have enabled to autofix spacing issues?

Reply:
![image](https://user-images.githubusercontent.com/2856501/61754606-c2ec9780-ad68-11e9-92b2-2a90e7cda9e1.png)

I use `Ctl + Alt + F`, I looked at enabling `prettier` on commit before, but it wasn't produce desirable formatting, although from this PR it looks like I'll have to look at that again becuase this seems be happening more and more where there are many distracting changes not related to the PR.